lib/extended_math.py

```diff
@@ -223,6 +233,11 @@ def arithmetic_mean(number):
         arity=1
     ),
 
+    "ÅU": MethodAttribute(
+        lambda sides, n: polygonal_number(ast_int_eval(sides), ast_int_eval(n)),
```

Comment:
Please let me know if there's a more idiomatic way to pass parameters here.

Reply:
I don't think there is another way. Perhaps something like placing the `ast_int_eval` in the method itself would increase readability, but it's not necessary as both options are quite readable (in my opinion).